pkg/generators/zync/api_deployment.go

```diff
@@ -36,7 +36,7 @@ func (gen *APIGenerator) deployment() func() *appsv1.Deployment {
 						}(),
 						Containers: []corev1.Container{
 							{
-								Name:  gen.GetComponent(),
+								Name:  "zync-api",
```

Comment:
this should be just `zync`, in the controlller code we added the **api** suffix to make it more easy to follow (distinguish between the api and worker component):
```suggestion
								Name:  "zync",
```